entries/c/cloudamqp.com.json

```
@@ -0,0 +1,17 @@
+{
+  "CloudAMQP": {
+    "domain": "cloudamqp.com",
+    "img": "cloudamqp-logo.png",
+    "tfa": [
+      "totp",
+      "sms"
+      
+    ],
+    "documentation": "https://www.cloudamqp.com/blog/two-factor-authentication.html",
+    "notes": "Sms is required as a backup method",
```

As SMS is an abbreviation for Short Messaging Service, all letters should be capitalized.
```suggestion
    "notes": "SMS is required as a backup method",
```